modules/installing-ocp-agent.adoc

```diff
@@ -133,43 +133,43 @@ IPv6 is supported only on bare metal platforms.
 
 . Create the `agent-config.yaml` file:
 +
-[source,yaml]
+[source,terminal]
 ----
-  cat > agent-config.yaml << EOF
-  apiVersion: v1alpha1
-  kind: AgentConfig
-  metadata:
-    name: sno-cluster
-  rendezvousIP: 192.168.111.80 <1>
-  hosts: <2>
-    - hostname: master-0 <3>
+$ cat > agent-config.yaml << EOF
```

Comment:
FYI this change (and those on lines 136, 60, and 58) were brought over from @bergerhoffer's feedback on the 4.13 implementation of this doc fix: https://github.com/openshift/openshift-docs/pull/63869#discussion_r1304719015